### PR TITLE
Create FrontEndTests101.md

### PR DESCRIPTION
**Pull request by annekainicUSDS**

### CODE REVIEW

File: docs/FrontEndTests101.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+There are 4 times tests can run:
+1. At any time during local development by running one of the npm scripts specified in `package.json`
+  - Any tests can be run during this time
+2. During the pre-commit hook
+  - Only linting is run automatically
+3. Before merging a branch to master via Jenkins
+  - Unit tests, e2e tests, and code coverage are automatically run during this time
+4. Before merging a branch to production via Jenkins
+  - Accessibility tests are automatically run during this time
+
```
Comment (ncksllvn) [inline docs/FrontEndTests101.md]: Right now the tests also run after merging to production. The devops team is planning on getting rid of the double test situation but not yet

**Review by ncksllvn — APPROVED** [review]
Looks great!

File: docs/FrontEndTests101.md (hunk 1 above)
Comment (annekainicUSDS) [reply docs/FrontEndTests101.md]: Ah good to know, thanks! Will update

Comment (ncksllvn) [reply docs/FrontEndTests101.md]: It adds some extra suspense during deployments lol. Sounds good

**Pull request merged**